hypha/apply/projects/tables.py

```diff
@@ -254,10 +255,10 @@ class Meta:
             "fund",
             "reporting",
             "last_payment_request",
-            "created_at",
+            "end_date",
         ]
         model = Project
         orderable = True
-        order_by = ("-created_at",)
+        order_by = ("-end_date",)
```

Comment:
```suggestion
        order_by = ("end_date",)
```
I think in their current spreadsheet staff sort by the ascending end dates so this may be a bit closer to that